shuup/notify/admin_module/views/editor.py

```diff
@@ -58,6 +58,8 @@ def script_item_editor(request):
         except ValidationError as verr:
             form.add_error(None, verr)
         else:
+            if "recipient" in form.script_item.data:
+                form.script_item.data['recipient']['constant'] = form.script_item.data['recipient']['constant'].pk
```

Comment:
Maybe would be better to identify how to put `pk` in `form.script_item.data['recipient']['constant']`directly so it comes in this point with proper value.. Although, I failed in trying to identify that point :)